draft-ietf-tls-dtls13.md

```diff
@@ -1556,7 +1568,8 @@ is to send out only part of the flight, sending more when
 messages are acknowledged. DTLS offers a number of mechanisms
 for minimizing the size of the certificate message, including
 the cached information extension {{?RFC7924}} and certificate
-compression {{?RFC8879}}.
+compression {{?RFC8879}}. DTLS stacks MUST NOT send flights
```

Comment:
The mailing list discussion recommended a SHOULD NOT, as far as I understand. It's not clear to me how a MUST NOT could be enforced in practice, if the e.g. the certificate chain is very long.

The issue here might be an overloading of the term "flight": As I read the paragraph, you use it not in the sense defined (https://www.ietf.org/archive/id/draft-ietf-tls-dtls13-41.html#name-dtls-handshake-flights) but in the sense of the part of the current flight which is currenly attempted to be transmitted. Using this latter definition, a MUST NOT as above could indeed be enforced, but ten then I think a different term should be used.

Reply:
Agreed. Fixed.